dex/networks/eth/contracts/ETHSwapV0.sol

```diff
@@ -199,10 +198,11 @@ contract ETHSwap {
     function refund(bytes32 secretHash)
         public
         senderIsOrigin()
-        isRefundable(secretHash)
     {
-        swaps[secretHash].state = State.Refunded;
-        (bool ok, ) = payable(msg.sender).call{value: swaps[secretHash].value}("");
+        require(isRefundable(secretHash));
+        Swap storage swapToRefund = swaps[secretHash];
```

Comment:
I don't expect it's significant, but I'm curious how refund gas costs change with this.  Refund is also not the usual swap outcome, so it's not critical, just wondering.

Reply:
This change actually saves a little gas, 42822 -> 42740.
The function call increased the gas, but using the local storage variable saved some.

Reply:
It doesn't help on isRedeemable because there are only 2 lookups there instead of the 3 here.